src/rtk/features/delegableApi/delegableApiSlice.ts

```diff
@@ -8,29 +8,51 @@ export interface DelegableApi {
   description: string;
 }
 
-export interface InitialState {
+export interface DelegableApiWithPriority {
+  id: string;
+  apiName: string;
+  orgName: string;
+  description: string;
+  priority: number;
+}
+
+export interface SliceState {
   loading: boolean;
   delegableApiList: DelegableApi[];
+  presentedApiList: DelegableApi[];
   chosenDelegableApiList: DelegableApi[];
+  delegableApiSearchPool: DelegableApi[];
+  apiSuppliers: string[];
```

Comment:
I think apiProviders is the english word for apiTilbydere

Reply:
Agreed 🙂